.github/workflows/Build.yml

```diff
@@ -35,19 +35,19 @@ jobs:
       - name: "Defining environment variables"
         if: startsWith(matrix.os, 'ubuntu')
         id: variablesLinux
-        run: echo "::set-output name=arg::$(if [ "${{ steps.files.outputs.global_files }}" = "" ] && ${{ github.event_name == 'pull_request' }}; then echo '--since'; else echo ''; fi)"
+        run: echo "arg=$(if [ "${{ steps.files.outputs.global_files }}" = "" ] && ${{ github.event_name == 'pull_request' }}; then echo '--since'; else echo ''; fi)" >> $GITHUB_OUTPUT
       - name: "Defining environment variables"
         if: startsWith(matrix.os, 'windows')
         id: variablesWindows
-        run: echo "::set-output name=arg::$(If ('${{ steps.files.outputs.global_files }}' -eq '' -AND '${{ github.event_name == 'pull_request' }}' -eq 'true'){ echo '--since' } Else { echo '' })"
+        run: echo "arg=$(If ('${{ steps.files.outputs.global_files }}' -eq '' -AND '${{ github.event_name == 'pull_request' }}' -eq 'true'){ echo '--since' } Else { echo '' })" >> $GITHUB_OUTPUT
```

Comment:
On windows you have to use `$env:GITHUB_OUTPUT`
https://docs.github.com/en/actions/learn-github-actions/environment-variables#detecting-the-operating-system